core-services/prow/02_config/_plugins.yaml

```diff
@@ -256,27 +256,6 @@ cherry_pick_unapproved:
   comment: This PR is not for the master branch but does not have the `cherry-pick-approved`  label.
     Adding the `do-not-merge/cherry-pick-not-approved`  label.
 config_updater:
-  cluster_groups:
-    build_farm_ci:
-      clusters:
-      - app.ci
-      - build01
-      - build02
-      - build03
-      - build04
-      - vsphere
-      namespaces:
-      - ci
-    build_farm_ocp:
-      clusters:
-      - app.ci
-      - build01
-      - build02
-      - build03
-      - build04
-      - vsphere
-      namespaces:
-      - ocp
```

Comment:
the changes in this _plugins.yaml file look a bit suspicious considering we're just trying to enable ci on the openshift/cluster-api-provider-kubevirt repo.

Reply:
Yes, also noticed to this change, created by the auto generator (make new-repo)
I removed this change